examples/python-guide/notebooks/interactive_plot_example.ipynb

```diff
@@ -237,7 +236,7 @@
FakiRJKsFALUmSJJVgoJYkSZJKMFBLkiRJJRioJUmSpBIM1JIkSVIJBmpJkiSpBAO1JEmSVIKBWpIkSSrBQC1JkiSVYKCWJEmSSjBQS5IkSSUYqCVJkqQSDNSSJElSCQZqSZIkqQQDtSRJklRCZGaza+i2iFgKLGx2HXqTEcALzS5Cb2JPWpN9aT32pDXZl9bTH3uyTWaOXN+kgb1RSQMszMw9ml2E/igi7rMnrcWetCb70nrsSWuyL63HnqybWz4kSZKkEgzUkiRJUgl9NVBf1uwC9Bb2pPXYk9ZkX1qPPWlN9qX12JN16JMvSpQkSZJaRV9doZYkSZJaQp8K1BFxQEQsjIjHI+LMZtfTX0XENyPi+Yh4pGZseETcERGPFd/f3cwa+5uIeF9E/GdE/DoifhkRf1mM25cmiYhNI+LeiHi46MnfF+Pvj4ifFT35TkRs3Oxa+6OIGBARD0bEzcV9+9JEEdEeEQsi4qGIuK8Y8/dXk0XEsIi4PiL+p/jvy972pWt9JlBHxADg/wEzgB2BoyNix+ZW1W99GzhgrbEzgTszczvgzuK+ek8ncHpmTgD2Av6i+P+HfWme14EPZ+YuwGTggIjYCzgP+FrRk5eA45tYY3/2l8Cva+7bl+ablpmTa96Wzd9fzXcxcFtm7gDsQvX/M/alC30mUAN7Ao9n5hOZ+QZwLXBIk2vqlzLzR8Dv1xo+BJhX3J4HfKJXi+rnMvOZzHyguL2U6i+9rbEvTZNVy4q7g4qvBD4MXF+M25MmiIgxwEHAFcX9wL60In9/NVFEbA7sB1wJkJlvZObL2Jcu9aVAvTWwqOb+4mJMrWHLzHwGquEOGNXkevqtiBgH7Ar8DPvSVMW2goeA54E7gN8AL2dmZzHF32PNcRHw18Cq4v57sC/NlsDtEXF/RJxYjPn7q7nGA0uAbxXbo66IiCHYly71pUAdXYz5FiVSjYgYCnwXODUzX2l2Pf1dZq7MzMnAGKp/ZZvQ1bTerap/i4iDgecz8/7a4S6m2pfetU9m7kZ1W+dfRMR+zS5IDAR2A76RmbsCy3F7xzr1pUC9GHhfzf0xwNNNqkVv9VxEjAYovj/f5Hr6nYgYRDVM/1tm3lAM25cWUPyZtEJ1f/uwiBhYHPL3WO/bB/h4RLRT3Tr4Yaor1valiTLz6eL788D3qP4D1N9fzbUYWJyZPyvuX081YNuXLvSlQP1zYLvildgbA0cB85tck/5oPjCruD0L+H4Ta+l3ij2gVwK/zsx/qjlkX5okIkZGxLDi9mDgI1T3tv8ncHgxzZ70ssz8UmaOycxxVP87cldmzsS+NE1EDImIzVbfBj4KPIK/v5oqM58FFkXE9sXQ/sCvsC9d6lMf7BIRB1JdSRgAfDMzz2lySf1SRFwDtAEjgOeAs4Ebgf8AxgK/Az6VmWu/cFENEhFTgR8DC/jjvtC/obqP2r40QURMovqCnQFUFy/+IzO/HBHjqa6MDgceBD6Tma83r9L+KyLagL/KzIPtS/MUP/vvFXcHAv+emedExHvw91dTRcRkqi/e3Rh4AjiW4vcZ9uVN+lSgliRJklpNX9ryIUmSJLUcA7UkSZJUgoFakiRJKsFALUmSJJVgoJYkSZJKMFBLUh0iYmVEPFTzNW4DrjEsIk7u+erWXP/jEdGrn2QWEZ+IiB178zElqdX4tnmSVIeIWJaZQ0teYxxwc2bu3M3zBmTmyjKP3QjFJwteQfU5Xd/seiSpWVyhlqQNFBEDIuKCiPh5RPwiIv68GB8aEXdGxAMRsSAiDilOORf4k2KF+4KIaIuIm2uu988RcUxxuz0i5kTEfwGfiog/iYjbIuL+iPhxROzQRT3HRMQ/F7e/HRHfiIj/jIgnIuJDEfHNiPh1RHy75pxlEXFhUeudETGyGJ8cET8tntf3IuLdxXglIr4SEXcDZwAfBy4ontOfRMSfFT+PhyPiuxHxrpp6/m9E/HdRz+E1Nfx18XN6OCLOLcbW+3wlqVUMbHYBktRHDI6Ih4rbT2bmocDxwB8y84MRsQlwT0TcDiwCDs3MVyJiBPDTiJgPnAnsnJmTYc0n9b2d1zJzajH3TuCkzHwsIqYAlwAfXs/57y7mfBy4CdgHOAH4eURMzsyHgCHAA5l5ekTMofrJp58H/hX4QmbeHRFfLsZPLa47LDM/VNS1HTUr1BHxcmZeXtz+x+Jn9PXivNHAVGAHqh9ffH1EzAA+AUzJzFcjYngx97INeL6S1BQGakmqz4rVQbjGR4FJNautWwDbAYuBr0TEflQ/Cn5rYMsNeMzvQHXFG/hT4LqIWH1skzrOvykzMyIWAM9l5oLier8ExgEPFfV9p5h/NXBDRGxBNTTfXYzPA65bu6512LkI0sOAocAPa47dmJmrgF9FxOqfx0eAb2XmqwCZ+fsSz1eSmsJALUkbLqiu4v7wTYPVbRsjgd0zsyMi2oFNuzi/kzdvvVt7zvLi+0bAy10E+vV5vfi+qub26vvr+v1fzwtrlr/NsW8Dn8jMh4ufQ1sX9UD1Z7f6+9qPuaHPV5Kawj3UkrThfgjMjohBABHxgYgYQnWl+vkiTE8DtinmLwU2qzn/t8COEbFJsSq8f1cPkpmvAE9GxKeKx4mI2KWHnsNGwOoV9k8D/5WZfwBeioh9i/HPAnd3dTJvfU6bAc8UP5OZdTz+7cBxNXuthzf4+UpSjzNQS9KGuwL4FfBARDwC/AvVld9/A/aIiPuohsr/AcjMF6nus34kIi7IzEXAfwC/KM558G0eayZwfEQ8DPwSOORt5nbHcmCniLif6h7lLxfjs6i+2PAXwOSa8bVdC3wxIh6MiD8BzgJ+BtxB8bzfTmbeRnU/9X3FHvW/Kg416vlKUo/zbfMkqR+LHng7QEnq71yhliRJkkpwhVqSJEkqwRVqSZIkqQQDtSRJklSCgVqSJP3/7daxAAAAAMAgf+tB7C2KgEGoAQBgEGoAABiEGgAAhgA3ESOFhO8zCgAAAABJRU5ErkJggg==",
```

Comment:
Did it just remove the line break at the end? That's a bit annoying, I don't think we have control on that